premake5.lua

```diff
@@ -165,6 +174,8 @@ workspace "slang"
     		
     filter { "system:linux" }
         linkoptions{  "-Wl,-rpath,'$$ORIGIN',--no-as-needed", "-ldl"}
+
```

Comment:
Why do we filter here for x86? I guess it doesn't do anything... but probably should be removed.

Reply:
I have no idea how that diff slipped in. Will remove.